src/Table/Body.tsx

```diff
@@ -1,12 +1,9 @@
 import React from "react";
-import TableSectionContext from "./util/tableSectionContext";
 
 const Body = ({ children }: { children: React.ReactNode }) => {
   return (
     <div className="nds-table-body" role="rowgroup">
-      <TableSectionContext.Provider value={{ section: "body" }}>
```

Comment:
This context was used for one thing only - telling `Cell` where in the table tree it lives by section (either "body" or "header").

Now that we no longer share `Cell` between the body and header of the table, we no longer need this context for conditional aria roles etc